dv/search_indexes.py

```diff
@@ -405,6 +405,13 @@ def prepare_financial_mechanism_ss(self, obj):
                 return [obj.project.financial_mechanism.grant_name]
         except Project.DoesNotExist:
             pass
+        if obj.programmes.exists():
+            pa = obj.programmes.values_list('programme_areas__pk', flat=True).distinct()
+            return list(Allocation.objects.filter(
```

Comment:
we should not need Allocation model here. This goes for organisation.prepare_financial_mechanism_ss as well.

the right way is via programme(or project).programme_area.priority_sector.type.name

Reply:
Programme too?

Reply:
Păi :)